docs/source/markdown/podman-run.1.md

```diff
@@ -1125,6 +1126,12 @@ container.
 You can add **:ro** or **:rw** option to mount a volume in read-only or
 read-write mode, respectively. By default, the volumes are mounted read-write.
 
+  `Chowning Volume Mounts`
+
+By default, Podman does not change the owner and group of source volume directories mounted into containers. If a container is created in a new user namespace, the UID and GID in the container may correspond to another UID and GID on the host.
+
+The `:U` suffix tells Podman to use the correct host UID and GID based on the UID and GID within the container, to change the owner and group of the source volume.
```

Comment:
This comment should mention that it will recursively change the UID of the files, and that this is potentially dangerous, and should be done with caution.

In SELinux we block certain volumes altogether from doing this.  Including $HOME.

Reply:
I have updated the docs and I have re used the relabel code for checking if the host path (source volume) can be modified.